src/swerve_description/urdf/parts/swerve_axle.xacro

```diff
@@ -2,33 +2,31 @@
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
 
     <!-- Chassis Properties -->
-    <xacro:property name="axle_length" value="0.075" />
-    <xacro:property name="axle_height" value=".1" />
-    <xacro:property name="axle_width" value="0.075" />
-    <xacro:property name="axle_weight" value="2.45" />
+    <xacro:property name="axle_radius" value="0.075" />
+    <xacro:property name="axle_length" value=".1" />
+    <xacro:property name="axle_mass" value="0.23739" />
+    <xacro:property name="wheel_radius" value="0.0508" />
+    <xacro:property name="wheel_length" value="0.0381" />
+    <xacro:property name="wheel_mass" value="0.23133211" />
 
     <!-- Wheel Properties -->
     <!-- NOTE: The below values are **guesses**-->
+    <!-- NOTE: Pretty good **guesses**-->
     <xacro:property name="wheel_pos_x" value="-0.022"/>
     <xacro:property name="wheel_pos_y" value="0.065"/>
 
     <xacro:macro name="swerve_axle" params="prefix parent *origin">
         <link name="${prefix}_axle_link">
-            <!-- TODO: Add intertia -->
             <visual>
                 <origin xyz="0 0 0" rpy="0 0 0" />
                 <geometry>
                     <mesh filename="package://swerve_description/meshes/SwerveAxle.stl" scale="0.001 0.001 0.001" />
                 </geometry>
                 <material name="blue" />
             </visual>
```

Comment:
Let's add a small collision property here. It will be used to assign mass and inertia to the axle.